src/main/java/se/uu/it/dtlsfuzzer/config/SulDelegate.java

```diff
@@ -15,35 +15,42 @@ public abstract class SulDelegate {
 	public static final String SUL_CONFIG = "/sul.config";
 	public static final String FUZZER_DIR = "fuzzer.dir";
 	public static final String SUTS_DIR = "suts.dir";
-	
+
 	@Parameter(names = "-protocol", required = false, description = "Protocol analyzed, determines transport layer used", converter = ProtocolVersionConverter.class)
 	private ProtocolVersion protocolVersion = ProtocolVersion.DTLS12;
 
 	@Parameter(names = "-timeout", required = false, description = "Time the SUL spends waiting for a response")
 	private Integer timeout = 100;
-	
+
 	@Parameter(names = "-inputResponseTimeout", required = false, description = "Time the SUL spends waiting for a response to a particular input. Expected format is: \"input1:value1,input2:value2...\" ", converter = InputResponseTimeoutConverter.class)
 	private InputResponseTimeoutMap inputResponseTimeout;
 
 	@Parameter(names = "-rstWait", required = false, description = "Time the SUL waits after executing each query")
 	private Long resetWait = 0L;
 
-	@Parameter(names = {"-command", "-cmd"}, required = false, description = "Command for starting the (D)TLS process")
+	@Parameter(names = { "-command",
```

Comment:
It's not clear why breaking `names` into two lines and adding a space after `{` or before `}` is better style.